cmapPy/pandasGEXpress/diff_gctoo.py

```diff
@@ -0,0 +1,38 @@
+'''
+diff_gctoo.py
+
+Given a GCToo object calculates differential values (expression, viability etc.)
```

Comment:
Maybe add: `Values can be made differential relative to all samples on a plate ("plate-control") or relative to negative control samples ("vehicle-control").`